core/src/main/java/org/apache/accumulo/core/client/BatchWriterConfig.java

```diff
@@ -309,6 +311,26 @@ public boolean equals(Object o) {
     return false;
   }
 
+  private static <T> T merge(T o1, T o2) {
+    if (o1 != null)
+      return o1;
+    return o2;
+  }
+
+  public BatchWriterConfig merge(BatchWriterConfig other) {
```

Comment:
This needs javadoc describing what it does and what it returns.  Also needs a since tag.

Reply:
Added in ce512a93bfa9